examples/documentation.html

```diff
@@ -327,7 +327,7 @@
 
         <div style="height: 50px"></div>
 
-		<h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>
+    <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>

```

Comment:
Well, the commit is obviously about the `wrapLabels` parameter as it's clear from the PR description. Changing indentation appeared here just by accident and turned out to be right thing.